int32 id
int32 parkingLotId
int32 type
int32 laneId
int32 pointId
int32 orderType
int32 side
geometry_msgs/Point[] points
